fulcrum/accessibility.html

```diff
@@ -143,6 +150,10 @@ <h3>Lever Press</h3>
 <p>Lever Press titles, while open access and freely available, are not available for download.</p>
 <p>To request remediation or a file download of any Lever Press title, send and email with the title of the content, your preferred format, and the URL to <a href="mpub-help@umich.edu">mpub-help@umich.edu</a>.</p>
 
+<h3>British Archaeological Reports Digital Collection</h3>
+<p><a href="/docs/Fulcrum-BAR-VPAT2.3INT-20200519.pdf">BAR Digital Collection Voluntary Product Accessibility Template (VPAT®) Version 2.3 INT</a></p>
+<p>To request remediation of any content in the UMP EBC, please send an email with the title of the content, your preferred format, and the URL to <a href="fulcrum-info@umich.edu">fulcrum-info@umich.edu</a>.</p>
```

Comment:
Is "UMP EBC" here (under BAR) intentional or copy-pasta?

Reply:
Nice catch. Yes, it was copy pasta. I've made the correction and pushed the updated branch.